nsot/conf/settings.py

```diff
@@ -290,6 +293,12 @@
 # Default: True
 NSOT_NEW_USERS_AS_SUPERUSER = True
 
+############
+# Guardian #
+############
+
+SILENCED_SYSTEM_CHECKS = ['guardian.W001']
```

Comment:
Can you please add a brief comment to explain this setting, too?

Reply:
Done.